assets/src/components/amp-support/index.js

```diff
@@ -0,0 +1,190 @@
+				} else {
+					setSubmitSupportRequest( false );
+					throw new Error( __( 'Failed to send support request. Please try again after some time', 'amp' ) );
+				}
+			} catch ( exception ) {
+				setError( exception.message );
+			} finally {
+				setSending( false );
+			}
+		} )();
+	}, [ submitSupportRequest, sending, restEndpoint, args ] );
+
+	return (
+		<div className="amp-support">
+			<Selectable>
+				<h2 className="amp-support__heading">
+					{ __( 'AMP Support', 'amp' ) }
+				</h2>
+				{ /* dangerouslySetInnerHTML reason: Injection of links. */ }
+				<p dangerouslySetInnerHTML={
+					{
+						__html: sprintf(
+							/* translators: %s is the URL to create a new support topic */
+							__( 'In order to best assist you, please submit the following information to our private database. Once you have done so, copy the the resulting support ID and mention it in a <a href="%s" rel="noreferrer" target="_blank">support forum topic</a>. You do not have to submit data to get support, but our team will be able to help you more effectively if you do so.', 'amp' ),
+							'https://wordpress.org/support/plugin/amp/#new-topic-0',
+						),
+					}
+				} />
+				<div className="amp-support__body">
+
+					{ data.site_info && <SiteInfo siteInfo={ data.site_info } /> }
+
+					{ data.themes && <Themes themes={ data.themes } /> }
+
+					{ data.plugins && <Plugins plugins={ data.plugins } /> }
+
+					{ data.errors && Array.isArray( data.errors ) && (
+						<Details
+							title={ sprintf(
+								/* translators: Placeholder is the number of errors */
+								__( 'Errors (%d)', 'amp' ),
+								data.errors.length,
+							) }
+							description={ __( 'Please check "Raw Data" for all error information.', 'amp' ) }
+						/>
+					) }
+
+					{ data.error_sources && Array.isArray( data.error_sources ) && (
+						<Details
+							title={ sprintf(
+								/* translators: Placeholder is the number of error sources */
+								__( 'Error Sources (%d)', 'amp' ),
+								data.error_sources.length,
+							) }
+							description={ __( 'Please check "Raw Data" for all error source information.', 'amp' ) }
+						/>
+					) }
+
+					{ data.urls && <ValidatedUrls validatedUrls={ data.urls } /> }
+
+					{ data && <RawData data={ data } /> }
+
+				</div>
+				<div className="amp-support__footer">
+					<Button
+						disabled={ Boolean( uuid ) || sending }
+						className="components-button--send-button is-primary"
+						onClick={ () => {
+							setSubmitSupportRequest( true );
+						} }
+					>
+						{ uuid && __( 'Sent', 'amp' ) }
+						{ sending && __( 'Sending…', 'amp' ) }
+						{ ( ! uuid && ! sending ) && __( 'Send data', 'amp' ) }
+					</Button>
+					{
+						uuid && (
+							<ExternalLink href="https://wordpress.org/support/plugin/amp/#new-topic-0">
+								{ __( 'Create support topic', 'amp' ) }
+							</ExternalLink>
+						)
+					}
+					{ error && (
+						<AMPNotice type="error" size="small">
```

Comment:
Let's import the `NOTICE_TYPE_ERROR` and `NOTICE_SIZE_SMALL` constants along with the `AMPNotice` itself and use them here instead of hardcoding the `type` and `size`:

```suggestion
						<AMPNotice
							type={ NOTICE_TYPE_ERROR }
							size={ NOTICE_SIZE_SMALL }
						>
```